app.js

```diff
@@ -41,7 +41,7 @@ app.use(cors());
 app.use(xss());
 app.use(mongoSanitize());
 
-//app.use(morgan('tiny')); //log  in terminal every request methode + statuscode
+app.use(morgan('combined')); //log  in terminal every request methode + statuscode
```

Comment:
not part of the review; I just changed this to see more details while testing out your code locally